schematools/types.py

```diff
@@ -316,6 +321,10 @@ def relations(self):
         """
         return self["schema"].get("additionalRelations", {})
 
+    @property
+    def auth(self):
+        """Auth of the table (if set)"""
+        return self.get("auth")
```

Comment:
hier zal ik al de `set` teruggeven, zodat de afnemer niet diverse JSON-notaties hoeft te parsen. Dat kan juist mooi ingekapseld worden in deze class